### ISSUE TITLE
Small comment on Anaconda Navigator

### ISSUE THREAD

**Issue by santisoler** [CodeToCommunicate/First_Semester_Course_Notebooks#12]
I saw on the slides that students will be instructed to run JupyterLab either from the shell or from Anaconda Navigator.

I'm more prone to normalize it to run it through the shell directly: I saw some issues with Anaconda Navigator in the past and since we expect them to get familiarized with the terminal, it's a good opportunity to use it for this purpose too.
Besides, Anaconda Navigator might not be available in some OS, so by avoiding using it, we normalize how we will do things during the sessions.

Feel free to share your thoughts!

**Comment by mdpiper** [comment]
@santisoler I totally agree--I won't cover Anaconda Navigator in the first lesson. We'll be using a JupyterHub, so it won't be an issue. We should probably mention it in one of the Saturday discussion sessions, though, just because it's not straightforward for people on Windows to access a shell.
